src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,121 @@
+package duke;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoField;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Storage;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+/**
+ * Encapsulates the main high-level logic of the bot.
+ */
```

Comment:
Maybe you can use the` /** header_comment_here*/` structure for the one-line header comments?